index.js

```diff
@@ -299,6 +303,10 @@ exports.collection = function (name, opts) {
 
     Schemas.schemas[name].fields = _.concat(opts.fields,
       Schemas.schemas[name].fields);
+  } else {
+    var collectionNames = _.map(Schemas.schemas, 'name');
+    logger.error('Cannot find a collection named "' + name + '" among [' +
+      collectionNames.join(', ') + '].');
```

Comment:
It's not a problem to have a collection that does not exist yet. Example: Smart collections